packages/livechat/src/routes/SwitchDepartment/connector.js

```diff
@@ -16,10 +16,10 @@ const SwitchDepartmentConnector = ({ ref, ...props }) => (
 					fontColor: customFontColor,
 					iconColor: customIconColor,
 				} = {},
+				guest,
```

Comment:
General note regarding SwitchDepartment component: Earlier this component was storing the department value in `state.department` property, however throughout the rest of the codebase, the department value was stored under `state.guest.department`. So this new change makes sure that this component follows the same pattern as the rest of the codebase & stores the department value in `state.guest.department`